module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3086,4 +3217,41 @@ protected function getItemsWithBibsForTransactions(
 
         return $items;
     }
+
+    /**
+     * Check if bib matches title hold rules
+     *
+     * @param array $bib    Bibliographic record fields
+     * @param array $patron An array of patron data
+     *
+     * @return bool True if request is valid, false if not
+     */
+    protected function checkTitleHoldRules(array $bib, array $patron): bool
+    {
+        if (!$this->titleHoldRules) {
+            return true;
+        }
+
+        if (
+            in_array('order', $this->titleHoldRules)
+            && !empty($bib['orders'])
+        ) {
+            return true;
+        }
+        if (
+            in_array('item', $this->titleHoldRules)
```

Comment:
Maybe this would be more readable if you assigned the in_array results to variables -- then you wouldn't have to search for 'item' twice, and you could incorporate the existence of the holdable_item rule into the if statement for isHoldable, which is not currently necessary but might make the code more explicit (and prevent possible bugs if more rules are added in future).

Reply:
Fixed.